src/config.ts

```diff
@@ -24,6 +24,7 @@
 
 import { Color, PaletteMode } from '@mui/material';
 import {
+    brown,
     cyan,
     deepPurple,
```

Comment:
```suggestion
    brown,
    cyan,
```

If we're not using deepPurple any more, we can presumably remove the import.